src/test/java/com/knighttodo/knighttodo/factories/TodoBlockFactory.java

```diff
@@ -19,6 +19,8 @@ private TodoBlockFactory(){}
     public static final String UPDATE_BLOCK_NAME = "for friday";
     public static final List<Todo> UPDATE_TODO_LIST = Arrays.asList(TodoFactory.updateTodo());
 
+
+
```

Comment:
those spaces are not needed here, leave your code well formatted during every commit.